testdata/p4_16_samples_outputs/inline-stack-bmv2-midend.p4

```diff
@@ -30,12 +30,9 @@ control ComputeChecksumI(inout H hdr, inout M meta) {
 }
 
 control IngressI(inout H hdr, inout M meta, inout std_meta_t std_meta) {
-    h_t[1] hdr_1_stack;
     @hidden action act() {
-        hdr_1_stack[0] = hdr.stack[0];
-        hdr.stack[0] = hdr_1_stack[0];
-        hdr_1_stack[0] = hdr.stack[0];
-        hdr.stack[0] = hdr_1_stack[0];
+        hdr.stack[0] = hdr.stack[0];
```

Comment:
These are no-ops. Do we have anything to detect this kind of code?

Reply:
Yes, `DoLocalCopyPropagation::postorder(IR::AssignmentStatement *)` is supposed to eliminate these -- I'll have to look into why it is missing this one.